packages/context/test/unit/resolver.test.ts

```diff
@@ -10,7 +10,14 @@ import {
   instantiateClass,
   invokeMethod,
   Injection,
+  Constructor,
+  RejectionError,
 } from '../..';
+import {ResolutionSession} from '../../src/resolver';
+
+function instantiate<T>(ctor: Constructor<T>, ctx: Context) {
+  return RejectionError.reject(instantiateClass<T>(ctor, ctx));
```

Comment:
👎 

The fact that `RejectionError` is polluting so many files is IMO yet another strong signal that we should look for a different approach (as I commented earlier).